firestore-bigquery-export/extension.yaml

```diff
@@ -315,7 +315,7 @@ params:
     label: BigQuery SQL table clustering
     description: >-
       This parameter will allow you to set up Clustering for the BigQuery Table
-      created by the extension. (for example: `data,document_id,timestamp`- no whitespaces). You can select up to 4 comma separated fields(order matters). 
+      created by the extension. (for example: `data,document_id,timestamp`- no whitespaces). You can select up to 4 comma separated fields. When you specify multiple columns, the order of the columns determines how the data is sorted. For example, if the table is clustered by columns a, b and c, the data is sorted in the same order: first by column a, then by column b, and then by column c. As a best practice, place the most frequently filtered or aggregated column first. 
```

Comment:
For easier review:: this is the only change made the others are generated by the format and generate-readmes

Reply:
Could we make this more concise? Something like `The order of the specified columns determines the sort order of the data` from the official [docs](https://cloud.google.com/bigquery/docs/clustered-tables)